src/executor/vsock.rs

```diff
@@ -0,0 +1,241 @@
+use alloc::collections::BTreeMap;
+use alloc::vec::Vec;
+use core::future;
+use core::task::{Poll, Waker};
+
+use endian_num::{le16, le32};
```

Comment:
We can use the types through `virtio-spec` instead:

```suggestion
use virtio::{le16, le32};
```

Reply:
solved it with my last commit